adcircpy/forcing/winds/best_track.py

```diff
@@ -830,18 +843,16 @@ def __file_end_date(self):
                 return date
 
     def __copy__(self) -> 'VortexForcing':
-        instance = self.__class__(
+        return self.__class__(
             storm=self.dataframe.copy(),
             start_date=self.start_date,
             end_date=self.end_date,
             file_deck=self.file_deck,
             record_type=self.record_type,
         )
-        instance.storm_id = self.storm_id
-        return instance
 
     def __eq__(self, other: 'VortexForcing') -> bool:
-        return numpy.all(self.dataframe == other.dataframe) and self.storm_id == other.storm_id
+        return numpy.all(self.dataframe == other.dataframe)
```

Comment:
why not import `numpy` like everyone else :)

Reply:
like `from numpy import all`?

Reply:
no like `import numpy as np`